RFC-0011-open-telemetry.md

```diff
@@ -0,0 +1,234 @@
+# **RFC0011 for Presto**
+
+
+## Enhancing Open Telemetry Implementation in Presto
+Proposers
+* Suresh Babu Areekara
+* Siddarth Ajay
+* Ben Tony Joe
+
+
+## [Related Issues]
+* https://github.com/prestodb/presto/issues/23975
+
+
+## Summary
+The existing Open Telemetry implementation https://github.com/prestodb/presto/pull/18534 was an experimental feature, had a limited set of telemetry data(Query state changes) and did not include a child span concept. The recent implementation will make Presto more flexible, allowing support for both parent and child spans. Additionally, traces can now be propagated to the worker nodes as well.
+
+
+## Background
+OpenTelemetry is a powerful serviceability framework that helps to gain insights into the performance and behaviour of the systems. It facilitates generation, collection, and management of telemetry data such as traces.
```

Comment:
We should drop the term "Open Telemetry" as much as possible from the document because we're trying to implement an SPI interface that open telemetry is capable of implementing. The SPI tracing provider interface is the important thing to highlight.